v2/src/features/UploadList/UploadList.tsx

```diff
@@ -52,11 +52,44 @@ const generateErrorPayload = (error: AxiosError | Error) => {
   return errorPayload;
 };
 
+const LOCAL_STORAGE_KEY = 'sugar-share-files';
```

Comment:
```suggestion
const LOCAL_STORAGE_KEY = 'sugarshare.files';
```
This is how cool kids name their storage keys

Reply:
https://github.com/sugarshare/sugarshare-extension-chrome/blob/master/v2/src/libs/auth.ts#L37